client/src/pages/Home.js

```diff
@@ -0,0 +1,64 @@
+import { useAuthenticationContext, useUserContext } from "libs/contextLib";
+import React from "react";
+import "../styles/Home.css";
+import SignInForm from "../components/auth/SignInForm";
+import { Table } from "react-bootstrap";
+
+const types = ["USER", "ADMIN", "MANAGER", "BOUNCER"];
```

Comment:
who's bouncer? I think we should go with the same types we have in nav.js (user, admin, manager, comms)